vanilla/bootstrap.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+if [ -z "$WORLD_FILENAME" ]; then
+    /terraria-server/TerrariaServer -world $@
```

Comment:
I havent tried to execute this code yet, but if I am reading this correctly, if the `$WORLD_FILENAME` is null, then why would we specify the world flag without a value?